### ISSUE TITLE
Resolve Floating Precision error

### ISSUE THREAD

**Issue by RasyidUFA** [svenhb/GRBL-Plotter#44]
as we can see, in the source, you used simple operation to comparing two floating point value

```csharp
if ((posPause.X != posWorld.X)){
    // this sometimes won'twork.
}
```
Comparison of floating point numbers can be unequal due to the differing precision of the two values.

then 

we can refactor this with

```csharp
// in some static class
public static double EPSILON= 1E-5; // 0.00001 Precision Point

// anywhere
if (Math.Abs(posPause.X - posWorld.X) > EPSILON){
    //This will always work
}
```

**Comment by svenhb** [comment]
It's a good point, I will take over this

**Comment by svenhb** [comment]
Implemented in Version 1.2.3.x, thanks for the hint
